### PR TITLE
Add user inventory dashboard

### PR DESCRIPTION
**Pull request by hardikhdholariya**
- Added the user inventory custom command to add, clean, merge and get the user details from lookup.
- Updated the saved searches related to user inventory.
- Added user inventory dashboard.
- Added macro on the cyences setup page.

### CODE REVIEW

File: cyences_app_for_splunk/bin/device_inventory_v2_util.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 Dev Details
-* It now stores everything inside a Python pickle file instead of a lookup
+* It now stores everything inside a lookup.
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/bin/device_inventory_v2_util.py:13]: inside a kvstore collection

**Review by VatsalJagani — APPROVED** [review]

File: cyences_app_for_splunk/default/macros.conf [hunk 2]
```diff
@@ -665,6 +665,14 @@ definition = "United States"
 definition = ".ad.crossrealms.com, .crossrealms.com, .local"
 # Put it with double quote on both side (compulsory)
 
+# User Inventory
+[cs_user_inventory_user_postfixes]
+definition = ".ad.crossrealms.com, .crossrealms.com, .local"
+# Put it with double quote on both side (compulsory)
+
+[cs_user_inventory_exclude_users]
+definition = NOT user IN ("*log4shell*", "$*")
```
[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:674]: username ends with $ has diff username as well, hence I've added @DC.REPUBLICOFTEA.COM to user_postfix macro to combine both.
image
Not sure but if it is not useful then we can remove it by adding *$ and *$@DC.REPUBLICOFTEA.COM to the user_exclude macro.

[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:674]: actually, It's for log4shell nessus users as 2 values are coming starting with $ but not have log4shell keyword in it.
image

[the remark this answers is not in the document]
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:674]: we can remove log4shell as $* covers everything.

[the remark this answers is not in the document]
Comment (VatsalJagani) [reply cyences_app_for_splunk/default/macros.conf:674]: got it.

**Pull request merged**